src/components/Layout/Layout.tsx

```diff
@@ -1,37 +1,61 @@
-import { useContext, useState } from 'react';
-import { Navigate, Outlet, useLocation } from 'react-router-dom';
-import AppNavigator from 'components/AppNavigator';
+import { useContext } from 'react';
+import { Outlet, Link as RouterLink } from 'react-router-dom';
+import Link from '@mui/material/Link';
+import StepIcon from '@mui/material/Step';
+import Stepper from '@mui/material/Stepper';
+import StepLabel from '@mui/material/StepLabel';
+import Typography from '@mui/material/Typography';
 import FormPersister from 'components/FormPersister';
-import routes from 'constants/routes';
 import WizardFormContext from 'contexts/WizardFormContext';
 import * as classes from './styles';
 
 export default function Layout() {
-  const [isInitialized, setIsInitialized] = useState<boolean>(false);
-  const { firstUncompletedStep } = useContext(WizardFormContext);
-  const { pathname } = useLocation();
-
-  if (pathname === routes.ROOT) {
-    return (
-      <Navigate
-        replace
-        to={routes[firstUncompletedStep]}
-      />
-    );
-  }
+  const { isInitialized } = useContext(WizardFormContext);
+  const { values, firstUncompletedStep, currentStep } =
+    useContext(WizardFormContext);
+  const activeStepIndex = currentStep ? values.indexOf(currentStep) : -1;
```

Comment:
Let's make it better: `currentStep && values.indexOf(currentStep) !== -1 ? values.indexOf(currentStep) : undefined;`